tests/www/test_views.py

```diff
@@ -488,7 +488,7 @@ def prepare_dagruns(self):
         )
 
     def test_index(self):
-        with assert_queries_count(42):
+        with assert_queries_count(43):
```

Comment:
Any idea what caused an extra query?

Reply:
It's the new menu link I added (based on me adding a second new link and it going up to 44). I don't know why FAB is doing a query per menu link, but that seems to be the behavior 🤷‍♂️.